rfc/rfc020-authorization-credential-extension.md

```diff
@@ -0,0 +1,64 @@
+# RFC020 Authorization credential extension
+
+|                           |                 |
+|:--------------------------|:----------------|
+| Nuts foundation           | W.M. Slakhorst  |
+| Request for Comments: 020 | Nedap           |
+| Ammends: RFC014           | April 2023      |
+
+## Authorization credential extension
+
+### Abstract
+
+A `assuranceLevel` field is added to the `NutsAuthorizationCredential`. It can be used within the context of a `resource` to indicate the required assurance level of the authentication.
```

Comment:
do we need a mapping of means to assuranceLevels (RFC019 = low, IRMA depends on actual DigiD-level)

Reply:
the current code already translates IRMA digid-level to assuranceLevel